server/schemas/resolvers.js

```diff
@@ -1,31 +1,136 @@
 // Desc: This file contains the graphql resolvers for the AdsTrees application.
```

Comment:
please remove the resolver file, because I have already added anything related to user and regarding preferences, Marko is working on that I'm not sure wht is he planning for